go.mod

```diff
@@ -46,6 +49,7 @@ require (
 	github.com/mdlayher/genetlink v0.0.0-20190313224034-60417448a851
 	github.com/mdlayher/netlink v0.0.0-20191009155606-de872b0d824b
 	github.com/mdlayher/raw v0.0.0-20190606144222-a54781e5f38f // indirect
+	github.com/mitchellh/go-wordwrap v1.0.0 // indirect
```

Comment:
where do the new deps come from?

Reply:
No clue... I was trying to figure that out. Modules are weird sometimes.

Reply:
I forced a revert.